CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 ## Current Develop Branch
 
 ## 8.1.1 Tue Oct 13 2020
+- [#9571](https://github.com/MetaMask/metamask-extension/pull/9571): Add data point to 'Swaps Completed' segment event: estimated vs used gas (#9571)
+- [#9586](https://github.com/MetaMask/metamask-extension/pull/9586): Hex prefix newly fetched token balances in build-quote (#9586)
```

Comment:
Here's an attempt to describe the impact this would have on a user:
```suggestion
- [#9586](https://github.com/MetaMask/metamask-extension/pull/9586): Prevent build quote crash when swapping from non-tracked token with balance (#9586)
```